src/Core/Silk.NET.Core.Win32Extras/Structs/IPrintDocumentPackageStatusEvent.gen.cs

```diff
@@ -261,190 +264,190 @@ public IPrintDocumentPackageStatusEvent
         }
 
         /// <summary>To be documented.</summary>
-        public readonly unsafe int Invoke(int dispIdMember, Guid* riid, uint lcid, ushort wFlags, DispatchParams* pDispParams, Variant* pVarResult, ExceptionInfo* pExcepInfo, uint* puArgErr)
+        public readonly unsafe int Invoke(int dispIdMember, Guid* riid, uint lcid, ushort wFlags, DispatchParams* pDispParams, Variant* pVarResult, ExcepInfo* pExcepInfo, uint* puArgErr)
```

Comment:
Why has this renamed?

Reply:
This one is due to `tagEXCEPINFO` being generated again, I have excluded it in win32extras.